### PR TITLE
Fixes #39

### LINKED ISSUE #39: Select: Add css classes to allow styling from external stylesheets
This should be a fairly straight forward change and should not effect the UI or functionality of the component. It's what we'd consider a "no harm" change. Here's what you need to do:

- [ ] Fork the repo
- [ ] run `npm install`
- [ ] run `npm run demo`. This will open up a new browser window with demos of each component. It will refresh whenever you save changes to the files in the `src` directory.
- [ ] open up `src/components/Select`
- [ ] On line 53, add `className='mx-select-scrim'` before the `onClick` prop declaration.
- [ ] On line 62 and 68, add `className='mx-select-options'` before the `style` prop declaration.
- [ ] On line 72, add `className='mx-select-option'` on it's own line.
- [ ] On line 91, add `className='mx-select'` before the `style` prop declaration.
- [ ] On line 93, add `className='mx-select-custom'` on it's own line before the `onBlur` prop declaration.
- [ ] On line 99, add `className='mx-select-selected'` before the `key` prop declaration.
- [ ] On line 111, add `className='mx-select-default'` before the `onChange` prop declaration.
- [ ] Open the demo and make sure your changes didn't break the Select component.
- [ ] run `eslint src` to check for syntax and style errors. Fix the errors as needed.
- [ ] Commit your changes to the `src/` directory
- [ ] Push your changes to your fork
- [ ] Create a PR and include `Fixes #39` in the message
- [ ] Get merged and celebrate :tada:

### PR DESCRIPTION
**Pull request by shubhekshajalan**

### CODE REVIEW

**Comment by bsbeeks** [comment]
`mx-select-scrim` should be placed on the element on this line: https://github.com/mxenabled/mx-react-components/pull/62/files#diff-37571e9f7a41053f949ea4cd1b82b0e9R53

**Comment by bsbeeks** [comment]
I realize there may have been some confusion because we forgot to update the line numbers in the description of issue #39 after updating the components to ES6 classes. I've updated the instructions with the correct line number. Please run through them again and make sure the css classes are on the correct elements. Also, there should only ever be 1 `className` defined per element. Let me know if you have any questions.

**Comment by shubhekshajalan** [comment]
Sorry about that. I should've asked earlier. I got confused as well due to the mismatched line numbers. Hopefully it's fixed now.

File: src/components/Select.js [hunk 1]
```diff
@@ -59,17 +59,17 @@ class Select extends React.Component {
...
         );
       } else {
         return (
-          <ul style={[styles.options, this.props.optionsStyle]}>
+          <ul className='mx-select-options' style={[styles.options, this.props.optionsStyle]}>
             {this.props.options.map(option => {
               return (
                 <li
-                  key={option.displayValue + option.value}
+                  className='mx-select-option' key={option.displayValue + option.value}
```
Comment (bsbeeks) [inline src/components/Select.js]: Move `key={option.displayValue + option.value}` to it's own line.

File: src/components/Select.js [hunk 2]
```diff
@@ -88,15 +88,15 @@ class Select extends React.Component {
     const selected = this.state.selected || this.props.selected || { displayValue: this.props.placeholderText, value: '' };
 
     return (
-      <div style={[this.props.style, { position: 'relative' }]}>
+      <div className='mx-select' style={[this.props.style, { position: 'relative' }]}>
         <div
-          onBlur={this._handleBlur.bind(this)}
+          className='mx-select-custom' onBlur={this._handleBlur.bind(this)}
```
Comment (bsbeeks) [inline src/components/Select.js]: Move `onBlur={this._handleBlur.bind(this)}` to it's own line

**Comment by bsbeeks** [comment]
Looks good! Congrats on your first open source PR, contribution! :tada:

**Comment by shubhekshajalan** [comment]
Yay, thanks! :smile:

**Pull request merged**